string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,16 @@
+/* exported swapChars */
+// Make storage for the output
+// Check to see where the firstIndex is then replace with the second index and store it
+// add the rest of the string before and after the secondIndex and store it
+// Return the storage as the output
+function swapChars(firstIndex, secondIndex, string) {
+  var revengeOfSith = '';
```

Comment:
Funny, but you should make names for your variables that describe their purpose. If a coworker needs to work with your code, they will **_vastly_** prefer understandable code over getting a chuckle, that's what comedians are for :joy: